torchtitan/experiments/qwen3/README.md

```diff
@@ -0,0 +1,21 @@
+**The Qwen3 model is still under development.**
+
+
+
+#### Available features
+QWEN3 0.6B Dense model - Other model sizes are also added to the args, but toml file configs need to be added and tested
```

Comment:
We have several different Qwen model flavors available, please update the description accordingly.